packages/@uppy/status-bar/types/index.d.ts

```diff
@@ -10,6 +10,10 @@ declare module StatusBar {
     showProgressDetails?: boolean
     hideUploadButton?: boolean
     hideAfterFinish?: boolean
+    hideRetryButton?: boolean,
+    hidePauseResumeButton?: boolean,
+    hideCancelButton?: boolean,
+    doneButtonHandler: Function | null,
```

Comment:
As is this would require users to pass in `doneButtonHandler: null`; we do need a `?` type here. Instead of `Function`, with typescript you can also use a function signature, so that users wont accidentally expect to receive any arguments:
```suggestion
    doneButtonHandler?: () => void,
```

Reply:
Oh sorry that's my bad. I forgot make it an optional type. Also wasn't sure what the function signature was supposed to be. Thanks